CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.0.2 under development
 
+- Bug #280: Fix type boolean (@terabytesoftw)
```

Comment:
Need change order of lines?

Reply:
Fix boolean type ?

Reply:
Yii Dev Tools in release process sort lines in change log automatically.